test/integration/puppetlabs/general_puppet/general_puppet_int_test.clj

```diff
@@ -259,7 +259,7 @@
 (deftest ^:integration static-file-content-endpoint-test-no-code-content-command
   (logging/with-test-logging
    (testing "the /static_file_content endpoint errors if :code-content-command is not set"
-     (bootstrap/with-puppetserver-running
+     (bootstrap/with-puppetserver-running-with-mock-jrubies
```

Comment:
I'm saying that we could add a fake docstring arg to the macro definition, so that consumers of the macro would be forced to pass in a string as the first argument.  The string could be ignored by the implementation of the macro but would enforce that we had to put a little snippet of text in each use case explaining why it's safe to circumvent the real rubies.  Does that seem like overkill?